apps/omg_child_chain_rpc/priv/swagger/operator_api_specs/block/request_bodies.yaml

```diff
@@ -4,6 +4,7 @@ GetBlockBodySchema:
   content:
     application/json:
       schema:
+        title: 'GetBlockBodySchema'
```

Comment:
Ehh, sorry about this - just noticed. This is only place we use apostrophes `'` - can you drop them, it's inconsistent to the rest of the file :cry:

Reply:
@pnowosie but we use single quotes in the rest of the yaml definition? https://github.com/omisego/elixir-omg/pull/1080/files/5ff5f142338455b23b3bbaefe21bea72c97c64dc#diff-1e0e669f0f7d082e7fa14f9ad3dc427eL15

Reply:
Well it looks like we use apostrophes in examples, but the rest goes without. Well if it does not make you a difference I'm for removing it there - it'll be simpler.